en-US/S.xml

```diff
@@ -346,6 +360,15 @@
 			</listitem>
 
 		</varlistentry>
+		 <varlistentry id="sign-on">
+			<term>sign-on, sign on</term>
+			 <listitem>
+				 <para>
+					Write hyphenated as an adjective or noun, such as "single sign-on", or as two words as a verb.
```

Comment:
Seeing these two entries next to each other, I think we should explain any preferences between them, as well as the preference between "log in", "sign in", and "sign on". Currently, we're presenting all three as viable options, but I'd prefer that we have more explicit guidance. For example, I imagine that we'd prefer "log in" in most cases, unless we need to align with a tool's UI or functionality (in which case, "sign in" or "sign on" are acceptable). (By way of example, Microsoft's "sign in" entry is very clear about what is and isn't allowed, and in what cases: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/sign-in-sign-out).